phoenix-core/src/main/java/org/apache/phoenix/coprocessor/IndexRebuildRegionScanner.java

```diff
@@ -1272,6 +1272,9 @@ private static Put prepareIndexPutForRebuid(IndexMaintainer indexMaintainer, Imm
             byte[] indexRowKey = indexMaintainer.buildRowKey(mergedRowVG, rowKeyPtr,
                     null, null, HConstants.LATEST_TIMESTAMP);
             indexPut = new Put(indexRowKey);
+        } else {
+            removeEmptyColumn(indexPut, indexMaintainer.getEmptyKeyValueFamily().copyBytesIfNecessary(),
```

Comment:
Can we add a comment here why we remove the empty column?

Reply:
There is no HBase client API to change the value of a cell. In order to change the value of the empty column cell, we need to remove it from the mutation and add a new cell with the new value (here the verify status). I will add a comment like this.